### PR TITLE
fix: don't output "ago" when container is currently up and running

### LINKED ISSUE #17250: [Bug]: HumanDuration STATUS of running container ("Up") should not contain "ago"
### Issue Description

In the following output from `podman ps`, the word "ago" should not be used when stating that the container is Up and running. It currently up and running, and has been up "1 second", **not** "1 second ago".

```
CONTAINER ID  IMAGE                            COMMAND     CREATED        STATUS                    PORTS       NAMES
fb20c6845f3b  docker.io/library/alpine:latest  /bin/sh     1 second ago   Up 1 second ago                       adoring_mestorf
```

### Steps to reproduce the issue

Steps to reproduce the issue
1. 
2.
3.

### Describe the results you received

Describe the results you received

### Describe the results you expected

Describe the results you expected

### podman info output

```shell
If you are unable to run podman info for any reason, please provide the podman version, operating system and its version and the architecture you are running.
```

### Podman in a container

No

### Privileged Or Rootless

None

### Upstream Latest Release

Yes

### Additional environment details

Additional environment details

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

### PR DESCRIPTION
**Pull request by quite**
Closes #17250

#### Does this PR introduce a user-facing change?

```release-note
Fix `podman ps` STATUS output to say "Up 3 seconds", if the container has been up and running for 3 seconds (removing the word "ago".
```

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks @quite 
/approve

Could you look at the man pages for ago, and remove it from them as well.

**Comment by quite** [comment]
@rhatdan good catch! fixed

**Review by vrothberg — APPROVED** [review]
LGTM, thank you!

**Comment by vrothberg** [comment]
Can you rebase your commit on the latest main branch? This will fix the test issue.

**Comment by vrothberg** [comment]
@containers/podman-maintainers PTAL

**Review by giuseppe — APPROVED** [review]
/lgtm
/hold

**Pull request merged**
